### PR TITLE
Don't return error if ingress already do not exist

### PR DESCRIPTION
**Pull request by panslava**
/assign @spencerhance @swetharepakula

### CODE REVIEW

File: pkg/e2e/helpers.go [hunk 1]
```diff
@@ -273,7 +273,7 @@ func performWhiteboxTests(s *Sandbox, ing *networkingv1.Ingress, fc *frontendcon
 // resources associated with it to be deleted.
 func WaitForIngressDeletion(ctx context.Context, g *fuzz.GCLB, s *Sandbox, ing *networkingv1.Ingress, options *fuzz.GCLBDeleteOptions) error {
 	crud := adapter.IngressCRUD{C: s.f.Clientset}
-	if err := crud.Delete(ing.Namespace, ing.Name); err != nil {
+	if err := crud.Delete(ing.Namespace, ing.Name); err != nil && !strings.Contains(err.Error(), "not found") {
```
Comment (spencerhance) [inline pkg/e2e/helpers.go:276]: let's use `if apierrors.IsNotFound(err)` rather than a substring check.

Comment (panslava) [reply pkg/e2e/helpers.go]: thanks, done

**Pull request merged**